dbms/src/TestUtils/ColumnsToTiPBExpr.cpp

```diff
@@ -179,6 +181,9 @@ void columnsToTiPBExpr(
     }
     else
     {
+        if (func_name == "grouping")
```

Comment:
I think `expr->set_val(val)` can always be called if `val` is not empty.

Reply:
> I think `expr->set_val(val)` can always be called if `val` is not empty.

okk, I will always call `expr->set_val(val)`